app/helpers/protocol_status_helper.rb

```diff
@@ -7,7 +7,7 @@ def protocol_status_href(protocol)
     res << 'data-trigger="focus" data-placement="bottom" title="'
     res << protocol_status_popover_title(parent) +
            '" data-content="' + protocol_status_popover_content(parent) +
-           '">' + protocol_name(parent) + '</a>'
+           '">' + protocol_name(parent).truncate(Constants::NAME_TRUNCATION_LENGTH) + '</a>'
     res.html_safe
```

Comment:
Don't you think this is a bit too short now? Also the acceptable size depends on the width of the window/screen.

Reply:
Well, it's default trunc length for most of fields. And if you click on it, you will see full name.